README.md

```diff
@@ -130,45 +130,10 @@ Performance][kubecon-cortex-201]" ([video][kubecon-cortex-201-video], [slides][k
 [promcon-2016-video]: https://youtu.be/3Tb4Wc0kfCM
 [promcon-2016-slides]: http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service
 
-## <a name="help"></a>Getting Help
-
-If you have any questions about Cortex:
-
-- Ask a question on the [Cortex Slack channel](https://cloud-native.slack.com/messages/cortex/). To invite yourself to the CNCF Slack, visit http://slack.cncf.io/.
-- <a href="https://github.com/cortexproject/cortex/issues/new">File an issue.</a>
-- Send an email to <a href="mailto:cortex-users@lists.cncf.io">cortex-users@lists.cncf.io</a>
+## Hosted Cortex
 
-Your feedback is always welcome.
-
-For security issues see https://github.com/cortexproject/cortex/security/policy
-
-## Community Meetings
-
-The Cortex community call happens every two weeks on Thursday, alternating at 1200 UTC and 1700 UTC. Meeting notes are held [here](https://docs.google.com/document/d/1shtXSAqp3t7fiC-9uZcKkq3mgwsItAJlH6YW6x1joZo/edit).
-
-To see meeting calendar:
-
-- See the calendar [in your browser (time zone will be UTC)](https://calendar.google.com/calendar/u/0/embed?src=cncf-cortex-maintainers@lists.cncf.io&ctz=UTC).
-- If you use Google Calendar, [add the Cortex's calendar to your own Google Calendar](https://calendar.google.com/calendar/u/0?cid=Y25jZi1jb3J0ZXgtbWFpbnRhaW5lcnNAbGlzdHMuY25jZi5pbw).
-- You can also just [download the .ics file](https://calendar.google.com/calendar/ical/cncf-cortex-maintainers%40lists.cncf.io/public/basic.ics).
```

Comment:
This is kind of important. why remove it?

Reply:
Agree. Especially if its current info for the community meeting.

Reply:
I've added back in the latest commit.